README.md

```diff
@@ -1,7 +1,7 @@
 # Calculated.gg Backend [![Build Status](https://api.travis-ci.org/SaltieRL/DistributedReplays.svg?branch=master)](https://travis-ci.org/SaltieRL/DistributedReplays) [<img src="https://img.shields.io/endpoint.svg?url=https%3A%2F%2Fshieldsio-patreon.herokuapp.com%2Fcalculated">](https://www.patreon.com/calculated)  [<img src="https://img.shields.io/discord/482991399017512960.svg?colorB=7581dc&logo=discord&logoColor=white">](https://discord.gg/c8cArY9)
 
 ## Setup
-- Install Python 3.6/pip
+- Install Python 3.6/pip (do not use Anaconda)
```

Comment:
We should have a *

And then below explain that celery will not function with anaconda as to explain why to not use anaconda